src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rearranged = {};
+  const robotValues = Object.values(robot);
+
+  const isValueUnique = new Set(robotValues).size === robotValues.length;
```

Comment:
You have a working solution, but let me ask you: do you really need to store in the memory one array and the set for getting info about unique values?
Or could you just check the keys in your new object?